tattoo_shop/data/ir_actions_server.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+  <record id="send_booking_email_server_action" model="ir.actions.server">
```

Comment:
Missing `template_id` and `mail_post_method` fields. As is, it won't do anything.